src/limits.js

```diff
@@ -3,12 +3,13 @@
  */
 var limits = {
     correctionRange: {
-        step: 1,
         max: {
             limit: 180,
             warning: 120,
+            noWarning: 121,
```

Comment:
Wouldn't `noWarning` be less than `warning` on `max`?

Reply:
doh .... yes it would! I will adjust